### DISCUSSION TITLE
Windows app to record time in programmes and quickly add them to Kimai

### DISCUSSION THREAD

**Discussion by EduNetEurope** [kimai/Paid-plugins#101] [Questions & Answers]
We are so happy we found Kimai. It is a great app.

The only thing we miss is a Windows app that records our time in Windows programmes and allows us to select projects / activities for the entries to automatically add them to Kimai. Clockify has such an app. Memtime unfortunately is not connected to Kimai at the moment.

**Comment by kevinpapst** [comment]
This repository is ONLY for posting topics that are related to paid plugins.
Please post here: https://github.com/kimai/kimai

**Reply by EduNetEurope** [reply]
Sorry!
